06-challenge/script.js

```diff
@@ -0,0 +1,16 @@
+function calcTip(bill) {
+  const tips = [0.15, 0.2];
+  if (bill >= 50 && bill <= 300) {
+    return bill * tips[0];
```

Comment:
if you had not chose to use the `tips` list this could have been `return bill * 0.15` which is clearer